Assets/Scripts/UI/DialogBox/DialogBoxNewGame.cs

```diff
@@ -29,6 +30,9 @@ public override void ShowDialog()
         Width.onEndEdit.AddListener(delegate { VerifyNumericInput(Width); });
         Depth.onEndEdit.AddListener(delegate { VerifyNumericInput(Depth); });
 
+        // Generate Random Seed TODO: make it alphanumerical
+        Seed.text = UnityEngine.Random.Range(0, int.MaxValue).ToString();
```

Comment:
Perhaps we could use a word-list, and randomly concatenate two or so of them, though of course that might be more something for another PR